tests/Kron.py

```diff
@@ -37,14 +39,111 @@
 
 #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
-class KronPhotometryTestCase(unittest.TestCase):
+def makeGalaxy(width, height, flux, a, b, theta, dx=0.0, dy=0.0, xy0=None, xcen=None, ycen=None):
```

Comment:
Sorry, I meant to say that makeGalaxy was part of the original code, and I don't think that I even looked at it.  Glad to try to make it better if you think it is worth the effort, but please put it on a different ticket.